web/src/lib/components/album-page/album-options.svelte

```diff
@@ -77,22 +123,40 @@
           </div>
           <div>{$t('invite_people')}</div>
         </button>
+        {#if currentUser}
         <div class="flex items-center gap-2 py-2 mt-2">
           <div>
-            <UserAvatar {user} size="md" />
+            <UserAvatar user={currentUser} size="md" />
           </div>
-          <div class="w-full">{user.name}</div>
+          <div class="w-full">{currentUser.name}</div>
           <div>{$t('owner')}</div>
         </div>
+        {/if}
+
         {#each album.albumUsers as { user } (user.id)}
           <div class="flex items-center gap-2 py-2">
             <div>
               <UserAvatar {user} size="md" />
             </div>
             <div class="w-full">{user.name}</div>
+            {#if user.id !== album.ownerId} <!-- Allow deletion for non-owners -->
+              <ButtonContextMenu icon={mdiDotsVertical} size="20" title={$t('options')}>
+                <MenuOption onClick={() => handleMenuRemove(user)} text={$t('remove')} />
+              </ButtonContextMenu>
+            {/if}
```

Comment:
Does it mean all users have that button ? You probably want to restrict that options for albums owners only. Currently there's 3 roles: the album owner, editor and viewer. We probably want to restrict that options for albums owner only. Though, it would probably nice to have a new role like album admin which has the same options as an album owner (but that's out of scope of this PR)

Reply:
As a sidenote: Just making the button available doesn't actually allow others to delete. These permissions are enforced serverside.